src/auto-evo/EditorAutoEvoRun.cs

```diff
@@ -6,10 +6,14 @@
 /// </summary>
 public class EditorAutoEvoRun : AutoEvoRun
 {
-    public EditorAutoEvoRun(GameWorld world, Species originalEditedSpecies, Species modifiedProperties) : base(world)
+    private readonly AutoEvoConfiguration configuration;
+
+    public EditorAutoEvoRun(GameWorld world, Species originalEditedSpecies, Species modifiedProperties,
+        AutoEvoConfiguration? configuration = null) : base(world, configuration)
     {
         OriginalEditedSpecies = originalEditedSpecies;
         ModifiedProperties = modifiedProperties;
+        this.configuration = configuration ?? SimulationParameters.Instance.AutoEvoConfiguration;
```

Comment:
The configuration is already passed to the base class on line 12, and unless I misremember this fallback logic already exists in there, so isn't this line unnecessary?